packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -123,18 +123,6 @@ module.exports = class Checkout extends Page {
         super.waitForComponent(this.errorPageComponent);
     }
 
-    isComponentDisplayed () {
-        return this.component.isDisplayed();
-    }
-
-    isFieldErrorDisplayed (fieldName) {
-        return this.fields[fieldName].error.isDisplayed();
-    }
-
-    isFieldDisplayed (fieldName) {
-        return this.fields[fieldName].input.isDisplayed();
-    }
-
     isPostcodeTypeErrorDisplayed () {
         return this.fields.addressPostcode.typeError.isDisplayed();
     }
```

Comment:
Did you want to move all the 'is displayed' tests? Is this one left here intentionally?

Reply:
This is intentional. There's a single component test left to check that error _isn't_ displayed when entering valid details. :)